Description: Use the Debian package name for pytest
Author: Michael R. Crusoe <crusoe@debian.org>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: pytest-mpi/tox.ini
===================================================================
--- pytest-mpi.orig/tox.ini	2022-02-01 14:04:35.909013227 +0100
+++ pytest-mpi/tox.ini	2022-02-01 14:04:47.293095939 +0100
@@ -8,8 +8,8 @@
 
 [testenv]
 commands =
-    oldpt: py.test -p pytester --ignore=docs --cov={envsitepackagesdir}/pytest_mpi --runpytest=subprocess {posargs}
-    !oldpt: py.test -p pytester --cov={envsitepackagesdir}/pytest_mpi --runpytest=subprocess {posargs}
+    oldpt: py.test-3 -p pytester --ignore=docs --cov={envsitepackagesdir}/pytest_mpi --runpytest=subprocess {posargs}
+    !oldpt: py.test-3 -p pytester --cov={envsitepackagesdir}/pytest_mpi --runpytest=subprocess {posargs}
 deps =
     pytest
     pytest-cov
@@ -31,6 +31,7 @@
     docs: {env:TOXPYTHON:python3}
     check-manifest: {env:TOXPYTHON:python3}
     checkreadme: {env:TOXPYTHON:python3}
+allowlist_externals=py.test-3
 
 [testenv:docs]
 changedir=docs
